packages/table/table-cell.scss

```diff
@@ -65,4 +59,22 @@ $expand-button-focus-color--inverted: jkl.$color-snohvit;
     .jkl-table--collapse-to-list[data-collapse] &::before {
         @include responsive-table-header;
     }
+
+    .jkl-table-row-expand-button {
+        display: flex;
+    }
+}
+
+.jkl-table-row--expandable.jkl-table-row--clickable {
```

Comment:
![Skjermbilde 2022-01-07 kl 12 14 39](https://user-images.githubusercontent.com/1223410/148536435-8dd7ec1d-fde6-47e7-86c1-3a07ae9d4fb9.png)

Listevisningen av denne er ikke helt 💯 
Får vi til en løsning hvor pila er inline med overskriften?

Reply:
Hmmja, det så ikke helt 🔥 ut nei. Dette er vel i grunn bare en problemstilling når man ikke har tekst sammen med pila

Reply:
Det tilfører også noen utfordringer ved at pil+data-th burde oppføre seg som én og samme knapp og da fullbredde med samme fokusmarkering for å gi et intuitivt og godt område å klikke på.

Reply:
Godt poeng med klikkbar label. Høres ut som du er på rett spor med mulig løsning 👍

Reply:
Tror jeg har fått til en kurant løsning nå, hvor komponenten sjekker om den er i listevisning, og hvis så prøver den å hente tekst fra data-th og putte inn i `ExpandButton`. Det føles som en relativt robust løsning som ivaretar brukeropplevelsen og tilgjengeligheten.

Reply:
Ser ut som et lovende spor 👍 

- Det ser ut som pila ikke er helt på linje med teksten med mindre man har `:hover` eller `:focus`. Spesielt synlig i listevisning. Virker typisk nok som det er noen pixelforskjeller mellom nettlesere – særlig Firefox. Muligens noe avrundingsgreier?
- På temaet tilgjengelighet, det minner meg på at siden vi bruker `ExpandButton` og ikke `ExpandSection` så må det noen ekstra attributter til både på knappen og på diven som ekspanderes: https://jokul.fremtind.no/komponenter/expandbutton#tilgjengelighet
- Relatert: jeg hadde forventet å kunne trykke både Enter og Space for å bruke knappen, men med Space virker det som et event går to ganger eller noe. Innholdet ekspanderer et _split second_ før det lukkes igjen. Enter er OK.

Reply:
Jeg la til litt mer attributter for tilgjengelighet, men jeg merket ikke all verdens forskjell med skjermleseren. Derimot la jeg merke til at Chrome+VO er veldig lite glad i tabeller inni tabeller. Men det fungerte fint i Safari.

Ang space har jeg fått den til å slutte med å lukke igjen, men jeg ser en forskjell i animasjonen når de åpnes som jeg ikke helt skjønner hvor kommer fra. Veldig rar oppførsel...

Reply:
Animasjonen ser ut til å fungere om fokuset er på selve raden i stedet for ExpandButton. To handlere som krangler kanskje?